ui/src/components/Switch/Switch.svelte

```diff
@@ -0,0 +1,43 @@
+<script>
+	let randomId = `toggle-${Math.random().toString(36).substring(2, 15)}`;
```

Comment:
Svelte has a feature for this that should be used instead https://svelte.dev/docs/svelte/$props#$props.id()